extensions/rich_text_components/Math/directives/oppia-noninteractive-math.directive.ts

```diff
@@ -36,8 +36,13 @@ angular.module('oppia').directive('oppiaNoninteractiveMath', [
       controller: ['$attrs', function($attrs) {
         var ctrl = this;
         ctrl.$onInit = function() {
-          ctrl.rawLatex = HtmlEscaperService.escapedJsonToObj(
-            $attrs.rawLatexWithValue);
+          ctrl.mathExpressionsContent = HtmlEscaperService.escapedJsonToObj(
+            $attrs.mathContentWithValue);
+          if (
+            ctrl.mathExpressionsContent !== undefined &&
+            ctrl.mathExpressionsContent.hasOwnProperty('raw_latex')) {
+            ctrl.rawLatex = ctrl.mathExpressionsContent.raw_latex;
```

Comment:
What if the above check failed, rawLatex will not be defined?

Reply:
modified